new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java

```diff
@@ -55,4 +62,50 @@ public static boolean doesFileExist(String fileName) {
     }
     return false;
   }
+  
+  /**
+   * Remove the contents of the given directory.
+   *
+   * @param dir the directory to be cleaned up
+   */
+  public static void cleanupDirectory(String dir) throws IOException {
+    File file = new File(dir);
+    logger.info("Cleaning up directory " + dir);
+    if (!file.exists()) {
+      // nothing to do
+      return;
+    }
+
+    if (!file.isDirectory()) {
+      throw new IllegalArgumentException("The parameter " + dir + " should be a directory.");
+    }
+    cleanDirectory(file);
```

Comment:
um what is going on here? this is an infinite recursion?

Reply:
hmm, I don't see recursiveness here. cleanDirectory is common-io FileUtils.cleanDirectory().  The method calls this is cleanupDirectory().

Reply:
And I tested the change:).

Reply:
oh sorry, my bad -- those names were so close :)